src/List/Filters.component.js

```diff
@@ -130,6 +129,8 @@ export default class Filters extends React.Component {
                 "dataViewOrganisationUnits.id": inFilter(orgUnitsOutput.map(ou => ou.id)),
             },
         };
+
+        return searchString ? { ...returnObj, query: searchString } : returnObj;
```

Comment:
Not that this is bad, but, for consistency, we can replicate the idea used with `showOnlyManagedUsers`, it's a typical pattern to conditionally set a key/value in an object. The other way is to do something like `_.omitBy(myObject, _.isNil)`, which removes any property that has null/undefined values. That's possible only when those values are not valid for this particular object, of course. Note that _.isNil = _.isNull + _.isUndefined, we can use any other predicate we want.